test/e2e/specs/specs-playwright/wp-theme__popover-preview.js

```diff
@@ -28,21 +28,22 @@ describe( DataHelper.createSuiteTitle( 'Theme: Preview' ), () => {
 		await sidebarComponent.gotoMenu( { item: 'Appearance' } );
 	} );
 
-	it( 'Search for free theme', async function () {
+	it( `Search for free theme with keyword ${ themeName }`, async function () {
 		themesPage = await ThemesPage.Expect( page );
 		await themesPage.filterThemes( 'Free' );
 		await themesPage.search( themeName );
 	} );
 
-	it( `Select ${ themeName }`, async function () {
-		await themesPage.select( themeName );
+	it( `Select ${ themeName } and click on Live Demo popover item`, async function () {
+		const selectedTheme = await themesPage.select( themeName );
+		await themesPage.clickPopoverItem( selectedTheme, 'Live Demo' );
 	} );
 
-	it( 'See theme detail page', async function () {
-		themesDetailPage = await ThemesDetailPage.Expect( page );
+	it( 'Preview theme', async function () {
+		previewComponent = await PreviewComponent.Expect( page );
 	} );
 
-	it( 'Preview theme', async function () {
-		await themesDetailPage.preview();
+	it( 'Close preview', async function () {
+		await previewComponent.closePreview();
```

Comment:
This file has been repurposed to test this flow:
- render themes gallery
- search for a clear theme name
- click on meatball menu
- trigger preview component from the popover